### PR TITLE
Add cron job to cleanup old work folders

### PR DESCRIPTION
**Pull request by xuqingfeng**
To solve disk get filled by old code.

### CODE REVIEW

File: tasks/cleanup.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+# cleanup old work folders to save disk space
+- name: Cleanup old work folders
+  cron:
+    name: cleanup old work folders
+    minute: 0
+    hour: 1
+    job: ls -ldt /var/lib/strong-pm/svc/1/work/* | tail -n +3 | xargs rm -rf
```
Comment (zbal) [inline tasks/cleanup.yml:8]: Would that exclude the `current` link from the list being cleaned?

**Review by zbal — COMMENTED** [review]
no objection - a simple question.

File: tasks/cleanup.yml (hunk 1 above)
Comment (xuqingfeng) [reply tasks/cleanup.yml:8]: `-d` will exclude symbolic links.

Comment (xuqingfeng) [reply tasks/cleanup.yml:8]: turns out `-d` won't exclude symbolic links but `ls` sort + `tail` will do the trick.
```
[root@staging-mobile-admin ~]# ls -ldt /var/lib/strong-pm/svc/1/work/*
lrwxrwxrwx  1 strong-pm strong-pm   84 Sep 12 19:17 /var/lib/strong-pm/svc/1/work/current -> /var/lib/strong-pm/svc/1/work/bd10ba332974d6c2d066337d2a4a00c0e4eccbfa.1505215015002
drwxr-xr-x 11 strong-pm strong-pm 4096 Sep 12 19:16 /var/lib/strong-pm/svc/1/work/bd10ba332974d6c2d066337d2a4a00c0e4eccbfa.1505215015002
---
[root@staging-mobile-admin ~]# ls -ldt /var/lib/strong-pm/svc/1/work/* | tail -n +3
[root@staging-mobile-admin ~]#
```

**Pull request merged**